src/styles/main.scss

```diff
@@ -0,0 +1,15 @@
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+
+  h2,
+  p {
+    margin: 0;
+  }
+}
```

Comment:
Consider moving general page styling into a separate file to keep your main file clean and clear